### DISCUSSION TITLE
[source-amplitude] The field size of the exported data is too large

### DISCUSSION THREAD

**Discussion by anchitbhatia** [airbytehq/airbyte#33478] [Connector Questions] [answered]
Currently i have an airbyte open source instance running locally in docker. I want to create Amplitude source. But when i am trying to test the source, it keeps failing with the attached error. I dont have any data in amplitude yet and using free version to test the integration. I tried with various start dates but i keep getting the same error. How can i fix this?
<img width="1008" alt="image" src="https://github.com/airbytehq/airbyte/assets/21287445/2314264d-fb6f-4954-813e-96a2ea58c6a8">

**Comment by marcosmarxm** [comment, marked as answer]
This is a limitation from [Amplitude API side](https://www.docs.developers.amplitude.com/analytics/apis/export-api/#considerations). Check the code implementation [here](https://github.com/airbytehq/airbyte/blob/577602554513da893477e5a2d093e25208bc9814/airbyte-integrations/connectors/source-amplitude/source_amplitude/manifest.yaml#L20-L22). The recommendation is to use a short start date parameter. Unfortunately the current connector doesn't support an `end_date` so you could manage the slices and would enables ingest a large historical record.
